aiohttp/web_protocol.py

```diff
@@ -630,7 +630,6 @@ async def finish_response(
         can get exception information. Returns True if the client disconnects
         prematurely.
         """
-        request._finish()
```

Comment:
@webknjaz Would be good if you can sanity check whether this method still makes sense. It was added only for this wait_for_disconnection() method, but then appears you added some multipart fix into the method. I'm wondering if that fix should live somewhere else, or we should keep the method.

Reply:
I assume your fix was never backported either: #5494.